protocol-definitions/MC.yaml

```diff
@@ -416,6 +416,12 @@ methods:
           since: 2.0
           doc: |
             Cluster-wide time in milliseconds.
+        - name: clusterId
```

Comment:
`ClusterId` is already retrieved by the client during authentication response and its type is UUID. See [here](https://github.com/hazelcast/hazelcast-client-protocol/blob/master/protocol-definitions/Client.yaml#L108) Why don't you just use it?

Reply:
I guess it's because it's a place where we don't have client authentication information, see https://github.com/hazelcast/management-center/blob/ab17d533efdb60de83ee4b38897d60a381a8285b/src/main/java/com/hazelcast/webmonitor/service/telemetry/ClusterUsageTracker.java#L76

Reply:
@ihsandemir is `clusterId` exposed from the Client? It seems it is not

Reply:
No, it is only stored in the TcpClientConnectionManager, and we don't expose it. It should be possible to expose it, but I am not sure if it is worth it

Reply:
I also checked `ClientAuthenticationCustomCodec#clusterId` in Platform, but the field doesn't seem to be passed anywhere

Reply:
I think it shouldn't be exposed as it has no meaning for customers. Therefore exposure in the private MC operation is a better option

Reply:
yes, it is not in the public API. Do you need the `clusterId` of the cluster that the MC client is connected and as seen by MC client? What about the clusterId of the other connected real clients?

Reply:
We need it only for MC, we are going to pass it to Phone Home to store in `callHome::mc_info` table.

Reply:
FYI I changed the type to UUID if that represents the field better

Reply:
to wrap up, shall we approve and move forward with this if we don't have any better option?

Reply:
I think the current approach is the simplest and the best one. @ihsandemir WDYT?

Reply:
One more remark, if we obtain it from the client itself, we can get this stat even for older version servers. See [this](https://github.com/hazelcast/hazelcast/pull/22280#discussion_r976701223)

Reply:
I assume we will miss 5.2 release if we take the way to expose it in client

Reply:
Since today we have to wrap up everything for 5.2.0

Reply:
If it is urgent for 5.2 release, we can not do the client changes that quick. Let's proceed.